sdk/tables/Azure.Data.Tables/api/Azure.Data.Tables.netstandard2.0.cs

```diff
@@ -15,6 +15,8 @@ public partial class TableClient
         public TableClient(System.Uri endpoint, string tableName, Azure.AzureSasCredential credential, Azure.Data.Tables.TableClientOptions options = null) { }
         public TableClient(System.Uri endpoint, string tableName, Azure.Data.Tables.TableSharedKeyCredential credential) { }
         public TableClient(System.Uri endpoint, string tableName, Azure.Data.Tables.TableSharedKeyCredential credential, Azure.Data.Tables.TableClientOptions options = null) { }
+        public string AccountName { get { throw null; } }
```

Comment:
virtual

Reply:
Thanks - Do you think the analyzer should be flagging these?

Reply:
I don't think we have an analyzer like that.

Reply:
I was thinking of the one that validates that we have a sync and async version of methods. I thought it also checked that they were virtual.